Automattic/jetpack
# My Jetpack: Enhance Product Card

Related with #29675 

## Proposed changes:

* Use a new sample design in the card (removing the description if it's in non-absent status).
* Support passing a `children` for `ProductCard` (it will be used next on providing stats in the VideoPress card).
* Extract `ActionButton` to own file in `ProductCard`.

### Other information:


## Jetpack product discussion


8IcdvSjekqRbBCf1xWmPOC-fi-3280%3A9537

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






_New Layout_

* Start `My Jetpack` and navigate to it page on `wp-admin`.
* You should see the new layout for non-absent cards.

_Children Support_

* Start `Storybook`
* Navigate to `My Jetpack/Components/Product Card`.
* You should see the new story `With Children`.
* 
## Demo

![CleanShot 2023-03-29 at 17 47 31](https://user-images.githubusercontent.com/1663717/228663699-9bf4d061-c64e-4980-b3ff-c00fd6237054.png)
